test/dotnet-resgen.Tests/Microsoft.DotNet.Tools.Resgen.Tests.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Microsoft.Extensions.PlatformAbstractions;
+using Xunit;
+using System;
+
+namespace Microsoft.DotNet.Tools.Resgen.Tests
+{
+    public class ResgenTests : TestBase
+    {
+        private readonly string _testProjectsRoot;
+        private readonly TempDirectory _root;
+
+        public ResgenTests()
+        {
+            _testProjectsRoot = Path.Combine(AppContext.BaseDirectory, "TestAssets", "TestProjects");
+            _root = Temp.CreateDirectory();
+        }
+
+        [Fact]
+        public void Test_Build_Project_with_Resources_with_Space_in_Path_Should_Succeed()
```

Comment:
Why do you say this is testing two lines of code? This is primarily testing that spaces in the path between the process chain involving resgen is handled correctly.